services/device-registry-base/src/main/java/org/eclipse/hono/service/management/device/Device.java

```diff
@@ -61,6 +61,10 @@
     @JsonFormat(with = JsonFormat.Feature.ACCEPT_SINGLE_VALUE_AS_ARRAY)
     private List<String> memberOf = new LinkedList<>();
 
+    @JsonProperty(RegistryManagementConstants.FIELD_STATUS)
+    @JsonInclude(value = Include.NON_EMPTY)
+    private Map<String, Object> status = new HashMap<>();
+
```

Comment:
My understanding is that the *status* is a generic concept which will most likely also be added to other entities (Tenant, Credentials), right? If so, I wonder if it deserves to be represented by a dedicated class instead of a map. In particular, since the type is defined to allow no additional properties in the API spec ...